python/examples/ingestion_with_python_config/rule_modules/velocity.yml

```diff
@@ -0,0 +1,12 @@
+namespace: velocity
+
+rules:
+  - name: vehicle_stuck
+    description: Checks that vehicle velocity becomes nonzero 5s after entering accelerating state
```

Comment:
It seems like the wording doesn't match the check. Should it say: "Checks that vehicle velocity is zero 5s after entering accelerating state"?

Same for below:
"Checks if vehicle velocity greater than 0 while stopped"

Reply:
The rule logic is sort of "backwards", so the condition evaluating to true means that it failed. So my thinking was if the vehicle velocity doesn't become nonzero while in the accelerating state then it must be stuck, therefore the rule should be that a vehicle velocity of 0 for 5s while in the accelerating state means the vehicle is stuck. But you're technically right... it is checking exactly what you said, in order to make sure the opposite happens 🤷 Lemme clarify these a bit